packages/asset-swapper/src/network/native_order_utils.ts

```diff
@@ -0,0 +1,47 @@
+import { BigNumber } from '@0x/utils';
+import { LimitOrderFields } from '@0x/protocol-utils';
+import { getContractAddressesForChainOrThrow } from '@0x/contract-addresses';
+
+import { SignedNativeOrder } from '../types';
+import { NativeOrderUtilsContract } from '../wrappers';
+
+import { Chain } from './chain';
+import { Address } from './types';
+import { ContractHelper, createContractWrapperAndHelper } from './utils';
+
+export class NativeOrderUtils {
```

Comment:
We can break up on-chain helpers into distinct classes/helpers that aren't aware of each other now because `ethCall()` auto-batches for us.